rfd/0127-resource-versioning.md

```diff
@@ -0,0 +1,34 @@
+---
+authors: Edoardo Spadolini (edoardo.spadolini@goteleport.com)
+state: draft
+---
+
+# RFD 0127 - Resource versioning
+
+## Required Approvers
+
+- Engineering: TBD ("@r0mant || @zmb3 || @russjones"?)
```

Comment:
I think that Product needs to sign off on this as well as this is technically a somewhat user-facing change.
```suggestion
- Engineering: TBD ("@r0mant || @zmb3 || @russjones"?)
- Product: @klizhentas || @xinding33
```